src/Cemu/DiscordPresence/DiscordPresence.cpp

```diff
@@ -1,7 +1,5 @@
 #include "DiscordPresence.h"
 
-#ifdef ENABLE_DISCORD_RPC
```

Comment:
Why was this removed?

Reply:
Because the files are now added or not added in CMake